spyql/spyql.py

```diff
@@ -226,57 +227,66 @@ def run(query):
     processor.go()
 
 
-def print_select_syntax():
-    print(
-        """
-  SELECT
-    [ * | python_expression [ AS output_column_name ] [, ...] ]
-    [ FROM csv | qy | text | arff | python_expression | json [ EXPLODE path ] ]
-    [ FILE path ]  ??
-    [ WHERE python_expression ]
-    [ LIMIT row_count ]
-    [ OFFSET num_rows_to_skip ]
-    [ TO csv | json | text | arff | py | sql | pretty | plot ]
+@click.command()
+@click.argument("query")
+@click.option(
+    "--profile",
+    "-p",
+    "profile",
+    is_flag=True,
+    default=False,
+    help=(
+        "Profile code with cProfile and writes output to the file spyql.stats."
+        " Collecting profiling statistics slows down query execution."
+    ),
+)
```

Comment:
I'm not sure if this should be included as a command-line option. This is a development tool not intended for use by the end-user and as such, it shouldn't be part of the public options.
Also, we're burning an option `-p` that could be useful for many things.

Reply:
Agree on not burning `-p`.
What about hiding `--profile` but keep it there for dev purposes?

Reply:
As discussed offline, this option will be removed.